[2023-09-27T17:06:19.246-0400] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_Server_Access_Log_Processing.extract manual__2023-09-27T21:06:06.163135+00:00 [queued]>
[2023-09-27T17:06:19.256-0400] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_Server_Access_Log_Processing.extract manual__2023-09-27T21:06:06.163135+00:00 [queued]>
[2023-09-27T17:06:19.256-0400] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2023-09-27T17:06:19.353-0400] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): extract> on 2023-09-27 21:06:06.163135+00:00
[2023-09-27T17:06:19.357-0400] {standard_task_runner.py:57} INFO - Started process 66666 to run task
[2023-09-27T17:06:19.360-0400] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'ETL_Server_Access_Log_Processing', 'extract', 'manual__2023-09-27T21:06:06.163135+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Server_Access_Log_Processing.py', '--cfg-path', '/tmp/tmp6_50_1ou']
[2023-09-27T17:06:19.361-0400] {standard_task_runner.py:85} INFO - Job 3: Subtask extract
[2023-09-27T17:06:19.497-0400] {task_command.py:415} INFO - Running <TaskInstance: ETL_Server_Access_Log_Processing.extract manual__2023-09-27T21:06:06.163135+00:00 [running]> on host erosennnin-TUF-Gaming-FX505DU-FX505DU
[2023-09-27T17:06:19.720-0400] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ramesh@somemail.com' AIRFLOW_CTX_DAG_OWNER='Ramesh Sannareddy' AIRFLOW_CTX_DAG_ID='ETL_Server_Access_Log_Processing' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2023-09-27T21:06:06.163135+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-27T21:06:06.163135+00:00'
[2023-09-27T17:06:19.722-0400] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-09-27T17:06:19.722-0400] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cut -f1,4 -d"#" web-server-access-log.txt > /home/erosennnin/Programing/Apache Airflow/dags/extracted.txt']
[2023-09-27T17:06:19.733-0400] {subprocess.py:86} INFO - Output:
[2023-09-27T17:06:19.736-0400] {subprocess.py:93} INFO - cut: web-server-access-log.txt: No such file or directory
[2023-09-27T17:06:19.736-0400] {subprocess.py:93} INFO - cut: Airflow/dags/extracted.txt: No such file or directory
[2023-09-27T17:06:19.736-0400] {subprocess.py:97} INFO - Command exited with return code 1
[2023-09-27T17:06:19.742-0400] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/erosennnin/anaconda3/envs/coursera/lib/python3.11/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-09-27T17:06:19.744-0400] {taskinstance.py:1398} INFO - Marking task as UP_FOR_RETRY. dag_id=ETL_Server_Access_Log_Processing, task_id=extract, execution_date=20230927T210606, start_date=20230927T210619, end_date=20230927T210619
[2023-09-27T17:06:19.864-0400] {standard_task_runner.py:104} ERROR - Failed to execute job 3 for task extract (Bash command failed. The command returned a non-zero exit code 1.; 66666)
[2023-09-27T17:06:19.892-0400] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-09-27T17:06:19.925-0400] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
